### PR TITLE
[AAE-6919] add a display text form control

### PR DESCRIPTION
**Pull request by alep85**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-6919

**What is the new behaviour?**
New component `adf-rich-text-editor` has been created to wrap [editorjs](https://editorjs.io/) tool and allows to write formatted text.
New form widget `display-rich-text` has been created to convert `adf-rich-text-editor` data to HTML

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by andrea-ligios — APPROVED** [review]
LGTM!

File: demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        ]
+    };
+
+    constructor() { }
+
+    ngOnInit(): void {
+    }
+
+    ngAfterViewInit(): void {
+        this.textEditor.outputData$.pipe(
```
Comment (mauriziovitale) [inline demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts:100]: don't think component should rely on property like that, better to use a service...

File: demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                data: {
+                    code: '// Amazing code example\n\ncatch(Exception ex){\n   // Houston, we have a problem\n}'
+                }
+            }
+        ]
+    };
+
+    constructor() { }
+
+    ngOnInit(): void {
```
Comment (mauriziovitale) [inline demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts:96]: empty method not needed

File: demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                id: 'xqqk0DEmqh',
+                type: 'code',
+                data: {
+                    code: '// Amazing code example\n\ncatch(Exception ex){\n   // Houston, we have a problem\n}'
+                }
+            }
+        ]
+    };
+
+    constructor() { }
```
Comment (mauriziovitale) [inline demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts:94]: why do u need it ?

Comment (alep85) [reply demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts:94]: I need it to show how the editor render the formatted text

File: demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts (hunk 1 above)
Comment (alep85) [reply demo-shell/src/app/components/rich-text-editor/rich-text-editor.component.ts:100]: You're right, should I create a bug or a new task to handle it by a service?

**Pull request merged**